k8s/helmfile/helmfile.yaml

```diff
@@ -198,9 +203,22 @@ releases:
     chart: ./../../charts/stats-cron
     <<: *default_release
 
+  - name: prometheus-elasticsearch-exporter
+    namespace: default
+    chart: prometheus-community/prometheus-elasticsearch-exporter
+    version: 5.0.0
+    <<: *default_release
+
   ################################
   # LOCAL ONLY
   ################################
+  - name: kube-prometheus-stack
```

Comment:
This *probably* wants a version pin (amazingly there was an update even today).

However one thing we are very bad is is then rolling forwards with these version updates